src/main/resources/archetype-resources/silo/src/main/resources/application.yml

```diff
@@ -45,3 +45,4 @@ remoteTokenServices:
 consul:
     name: ${spring.config.name}
     agent: 192.168.33.10
+    siloPort: ${server.port}
```

Comment:
Whoot? https://github.com/rebuy-de/rebuy-silo-archetype/pull/30/files#diff-51c32271e32ab3de48175118fbc19360R48 you just have to say "port" instead of siloPort..